group.py

```diff
@@ -1,5 +1,34 @@
-"""An example of how to represent a group of acquaintances in Python."""
-
-# Your code to go here...
-
-my_group =
+my_group = {
+    'Jill' : {
+        'relationships': {

```

Comment:
Interesting method for generalizing relationships of any type. Will be very exstensible in the future. However, this you will run into problems when a person has more than one relationship with a single person (e.g. Zalika-friend, Zalika-cousin).